src/components/Badge/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { ComponentProps, ReactElement, ReactNode,forwardRef } from "react";
+import clsx from "clsx";
+import { TGenericSizes } from "../../types";
+import { Text } from "../Text";
+import "./Badge.scss";
+
+type TVariant = "contained" | "bordered";
+type TColor =
+    | "blue"
+    | "light-blue"
+    | "general"
+    | "purple"
+    | "success"
+    | "warning"
+    | "danger"
+    | "gold"
+    | "green"
+type TPadding = "tight" | "loose";
+
+interface BadgeProps extends ComponentProps<"div"> {
+    badgeSize?: Extract<TGenericSizes, "lg" | "md" | "sm" | "xs">;
+    children: ReactNode;
+    color?: TColor;
+    isBold: boolean;
```

Comment:
```suggestion
    isBold?: boolean;
```

Is it expected to make `isBold` a required prop?

Reply:
I am making it optional.Thanks